js/test/release.test.ts

```diff
@@ -10,19 +10,6 @@ describe.skip('Integration', () => {
   const path = `m/44'/1234'/0'/0/0`
 
   // enable this for integration test
-  it('should test node', async () => {
```

Comment:
Let's skip this test by default, instead of removing it. I used this to test with my physical ledger

Reply:
I removed this test because we removed the `SignHash` instruction. I can add a test case for `SignUnsignedTx`, but this requires running the Alephium devnet. Alternatively, we could still keep the `SignHash` instruction. WDYT?

Reply:
Let's test `SingnUnsignedTx` with devnet